### PR TITLE
Implement 404 page

### PR DESCRIPTION
**Pull request by MehulKChaudhari**
Fixes: https://github.com/ember-learn/ember-api-docs/issues/884

Desktop: 
image

Mobile: 
image

### CODE REVIEW

**Comment by MehulKChaudhari** [comment]
Hey @MinThaMie, Please have a look at this PR? I have made changes keeping `website-redesign` as base branch.

**Comment by evoactivity** [comment]
Could I suggest something more like this

image

image

---

1) Add padding to the `.whoops` class, my example screenshots are `4rem` on desktop and `2rem` on mobile
2) remove `flex-direction: column;` on desktop
3) reduce size of tomster to around `240px` on desktop

**Comment by MinThaMie** [comment]
I wholeheartedly agree with @evoactivity  :) If you implement those changes I'll approve :)

**Comment by MehulKChaudhari** [comment]
> I wholeheartedly agree with @evoactivity :) If you implement those changes I'll approve :)

Made the changes, but Instead of `2rem` padding in mobile I have used `--spacing-3` variable which is `1.5rem` does that work or we should add hard coded `2rem` value? 
@MinThaMie @evoactivity

**Comment by MinThaMie** [comment]
I'm happy with the `--spacing-3`

**Review by mansona — DISMISSED** [review]
one last thing and it should be good to go 👍

File: tests/unit/routes/not-found-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | not-found', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    let route = this.owner.lookup('route:not-found');
+    assert.ok(route);
+  });
+});
```
[the remark this answers is not in the document]
Comment (MinThaMie) [reply tests/unit/routes/not-found-test.js]: Fixed this for you to keep the speed of this PR going :) @MehulKChaudhari thanks for picking this up and fixing it so quickly :)

[the remark this answers is not in the document]
Comment (MehulKChaudhari) [reply tests/unit/routes/not-found-test.js]: Thanks for deleting the Test :). Looking for more tasks.

**Review by MinThaMie — APPROVED** [review]

**Comment by MehulKChaudhari** [comment]
> Thank you very much @MehulKChaudhari

Thank you, Please let me know if there is any task that I can pick up.

**Pull request merged**